tests/sync/questionsSync.js

```diff
@@ -77,6 +79,8 @@ describe('Question syncing', () => {
     // We'll just sync the same course from two different directories.
     // Since courses are identified by directory, this will create two
     // separate courses.
+    // TODO: we should actually assert that two independent copies of all data exist.
+    // While developing, I found some cases where data wasn't correctly scoped to one course/course instance.
```

Comment:
I assume this TODO and the ones below are temporary, but flagging them here just in case.